src/cli/utils.js

```diff
@@ -49,3 +49,15 @@ exports.getIPFS = (callback) => {
 exports.getRepoPath = () => {
   return process.env.IPFS_PATH || os.homedir() + '/.ipfs'
 }
+
+exports.createLogger = (visible) => {
+  return (msg, newline = true) => {
+    if (visible) {
+      if (msg === undefined) {
+        msg = ''
+      }
+      msg = newline ? msg + '\n' : msg
+      process.stdout.write(msg)
+    }
+  }
+}
```

Comment:
Isn't this just a console.log? All invisible logs should be through `debug` (http://npmjs.org/debug)